src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -31,6 +36,7 @@ class FileOutputWriter(props: Map[String, Object], outputFile: Option[File]) ext
   override def write(dataFrame: DataFrame): Unit = {
     val writer = dataFrame.write
 
+    val currentTimestamp = DateTime.now.toString("YYY-MM-dd--HH-mm-ss")
```

Comment:
I like unixtime better, but it's not critical

Reply:
I thought we wanted it to be as in Airflow, unix time is easier but less readable, wdyt?

Reply:
I meant for the path to look the same (in terms of where to place the is in the prefix). Again, as you like :)